month_end/models/month_end.py

```diff
@@ -0,0 +1,125 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2017 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: Maxime Beck
+#
+#    The licence is in the file __openerp__.py
+#
+##############################################################################
+
+import datetime
+import calendar
+from odoo import api, models, fields
+
+
+class MonthEnd(models.Model):
+    _name = 'month.end'
+
+    ##########################################################################
+    #                             FIELDS METHODS                             #
+    ##########################################################################
+    @api.model
+    def process_create_accounting_entry(self):
+        date_format = '%d.%m.%Y'
+
+        today = datetime.date.today()
+        first_day = today.replace(day=1)
+        last_day = today.replace(day=calendar.monthrange(today.year, today.month)[1])
+
+        first_day_last_month = first_day.replace(month=today.month-1)
+        last_day_last_month = last_day.replace(month=today.month-1)
```

Comment:
Shouldn't you use the `monthrange` method here as well to ensure you find the last day of the month?